gnome-settings/setup-gnome-tweaks.sh

```
@@ -0,0 +1,416 @@
+    log_info "⌨️ キーバインド設定を適用中..."
+    
+    # Shell全般のキーバインド
+    apply_dconf_setting "/org/gnome/shell/keybindings/toggle-message-tray" "@as []" "メッセージトレイトグル無効"
+    apply_dconf_setting "/org/gnome/shell/keybindings/toggle-overview" "['<Primary><Alt>Tab']" "オーバービュートグル"
+    
+    log_success "🎉 キーバインド設定の適用が完了しました！"
+}
+
+# バックアップ作成関数
+create_backup() {
+    local backup_dir="$HOME/.config/gnome-settings-backup-$(date +%Y%m%d_%H%M%S)"
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)